reader-osm/src/main/java/com/graphhopper/reader/osm/GraphHopperWriterOSM.java

```diff
@@ -0,0 +1,16 @@
+package com.graphhopper.reader.osm;
+
+import com.graphhopper.GraphConfig;
+import com.graphhopper.GraphHopperWriter;
+import com.graphhopper.routing.util.EncodingManager;
+
+public class GraphHopperWriterOSM {
+    private GraphHopperWriterOSM() {
+    }
+
+    public static GraphHopperWriter create(EncodingManager encodingManager, GraphConfig graphConfig) {
+        GraphHopperWriter writer = new GraphHopperWriter(encodingManager, graphConfig);
+        OSMReader reader = new OSMReader(writer.getGraphHopperStorage());
+        return writer.setDataReader(reader);
```

Comment:
I really like the idea we no longer need to use an extra `GraphHopperOSM` if we want to do an OSM import. Maybe instead of setting the data reader we could even have something like `hopper.read(osmReader)`?